### PR TITLE
Add sweeps for bitwise ops

### PR DESCRIPTION
**Pull request by amalbasaTT**
### Ticket
https://github.com/tenstorrent/tt-metal/issues/11512

### Problem description
Missing sweeps in new sweeps infrastructure for bitwise ops:
1. bitwise_left_shift
2. bitwise_right_shift
3. bitwise_xor
4. bitwise_or
5. bitwise_and
6. bitwise_not

### What's changed
Added .py sweeps to new sweeps infrastructure for the above mentioned eltwise ops. 
Modified .github/workflows/ttnn-run-sweeps.yaml file to include above mentioned eltwise ops.

### Checklist
- [x] ttnn sweeps CI passes 
- [x] Post commit CI passes
- [x] Blackhole Post commit (if applicable)
- [x] Model regression CI testing passes (if applicable)
- [x] Device performance regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

File: tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_and_.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    input_tensor_a = ttnn.from_torch(
+        torch_input_tensor_a,
+        dtype=input_a_dtype,
+        layout=input_a_layout,
+        device=device,
+        memory_config=input_a_memory_config,
+    )
+
+    start_time = start_measuring_time()
+    result = ttnn.bitwise_and(input_tensor_a, value=scalar, memory_config=output_memory_config)
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_and_.py:80]: is this supposed to be `bitwise_and_` ?

File: tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_left_shift_.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    input_tensor_a = ttnn.from_torch(
+        torch_input_tensor_a,
+        dtype=input_a_dtype,
+        layout=input_a_layout,
+        device=device,
+        memory_config=input_a_memory_config,
+    )
+
+    start_time = start_measuring_time()
+    result = ttnn.bitwise_left_shift(input_tensor_a, shift_bits=shift_bits, memory_config=output_memory_config)
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_left_shift_.py:105]: Same question as above?

File: tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_not_.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    input_tensor_a = ttnn.from_torch(
+        torch_input_tensor_a,
+        dtype=input_a_dtype,
+        layout=input_a_layout,
+        device=device,
+        memory_config=input_a_memory_config,
+    )
+
+    start_time = start_measuring_time()
+    result = ttnn.bitwise_not(input_tensor_a, value=scalar, memory_config=output_memory_config)
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_not_.py:82]: Same as above?

File: tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_or_.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    input_tensor_a = ttnn.from_torch(
+        torch_input_tensor_a,
+        dtype=input_a_dtype,
+        layout=input_a_layout,
+        device=device,
+        memory_config=input_a_memory_config,
+    )
+
+    start_time = start_measuring_time()
+    result = ttnn.bitwise_or(input_tensor_a, value=scalar, memory_config=output_memory_config)
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_or_.py:80]: Same

File: tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_right_shift_.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    input_tensor_a = ttnn.from_torch(
+        torch_input_tensor_a,
+        dtype=input_a_dtype,
+        layout=input_a_layout,
+        device=device,
+        memory_config=input_a_memory_config,
+    )
+
+    start_time = start_measuring_time()
+    result = ttnn.bitwise_right_shift(input_tensor_a, shift_bits=shift_bits, memory_config=output_memory_config)
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_right_shift_.py:80]: Same question

File: tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_xor_.py [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+    input_tensor_a = ttnn.from_torch(
+        torch_input_tensor_a,
+        dtype=input_a_dtype,
+        layout=input_a_layout,
+        device=device,
+        memory_config=input_a_memory_config,
+    )
+
+    start_time = start_measuring_time()
+    result = ttnn.bitwise_xor(input_tensor_a, value=scalar, memory_config=output_memory_config)
```
Comment (jdesousa-TT) [inline tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_xor_.py:80]: Same

**Review by jdesousa-TT — CHANGES_REQUESTED** [review]
Can you please double check if these tests are supposed to be the inplace versions of these ops? i.e. with the `_`? Or if they are not, can you make the file names consistent with the actual op they test?

File: tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_and_.py (hunk 1 above)
Comment (amalbasaTT) [reply tests/sweep_framework/sweeps/eltwise/unary/bitwise/bitwise_and_.py:80]: No, problem is when performing query on Kibana, that "and" at the end is conflicting with the keyword "and" used in Kibana query language.

**Pull request closed without merging**